resources/import.sql

```diff
@@ -78,11 +79,21 @@ CREATE TABLE IF NOT EXISTS npwd_sellout_listings (
   PRIMARY KEY (id)
 )
 ENGINE = INNODB,
-AUTO_INCREMENT = 26,
+AUTO_INCREMENT = 31,
 AVG_ROW_LENGTH = 2048,
 CHARACTER SET utf8mb4,
 COLLATE utf8mb4_general_ci;
 
+CREATE TABLE IF NOT EXISTS npwd_marketplace_reports (
```

Comment:
This isn't strictly necessary but I found that when apps get to production having these two columns:

```
  `createdAt` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `updatedAt` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

In every table add a ton of debugging/troubleshooting/features/general helpfulness. Basically when you create a new row both columns are set to the current time. If you ever perform updates to a row `updatedAt` is set to the new time.

It requires no changes to SQL queries and sort of just maintains itself.